src/main/de/linearbits/preferences/PreferenceBoolean.java

```diff
@@ -37,11 +51,20 @@ public PreferenceBoolean(String label) {
 
     @Override
     protected Editor<Boolean> getEditor() {
-        return new EditorBoolean(getDialog(), getDefault());
+        return new EditorBoolean(getDialog(), getDefault(),this.hideLabel);
     }
 
     @Override
     protected Validator<Boolean> getValidator() {
         return null;
     }
-}
+
+    public Boolean getHideLabel() {
```

Comment:
Two comments:

(1) The method is missing a JavaDoc comment.
(2) The method is not needed.

-> Please remove.